applications/crossbar/doc/ref/cdrs.md

```diff
@@ -72,6 +72,16 @@ curl -v -X GET \
 
 #### Fetch
 
+> GET /v2/accounts/{ACCOUNT_ID}/cdrs/summary
```

Comment:
Looks like a new endpoint path that should be added to the [doc](https://github.com/2600hz/kazoo/blob/master/applications/crossbar/doc/cdrs.md), eh?